### PR TITLE
Coveralls

### PR DESCRIPTION
**Pull request by srl295**
Add https://coveralls.io testing, modelled  after [strong-globalize](https://github.com/strongloop/strong-globalize/blob/master/.travis.yml)

### CODE REVIEW

**Comment by srl295** [comment]
depends on #42

File: .travis.yml [hunk 1]
```diff
@@ -3,3 +3,7 @@ node_js:
   - "node"
   - "6"
   - "0.12"
```
Comment (Setogit) [inline .travis.yml]: why not v4?  v4 is an active LTS.

**Comment by Setogit** [comment]
LGTM

File: .travis.yml (hunk 1 above)
Comment (srl295) [inline .travis.yml]: "Node" is 4. But anyways it's a copy of yours now, minus 0.10 which broke for me.

**Comment by yumaoka** [comment]
LGTM

File: .travis.yml (hunk 1 above)
Comment (srl295) [inline .travis.yml]: https://docs.travis-ci.com/user/languages/javascript-with-nodejs - "node: latest stable Node.js release"

**Pull request merged**